python/hyperspace/includedcolumns.py

```diff
@@ -0,0 +1,12 @@
+class IncludedColumns:
```

Comment:
Do we have a better name for this? How about `IndexColumnConfig` or `IndexConfig.Columns` ?
It looks bit confusing because of "exclude"

Reply:
IncludedColumns -> IncludedColumnsConfig